index.html

```diff
@@ -1,278 +1,375 @@
-                     feature or website. Take a look at my work.</p><br>
-                <p class="intro_para"> If you like what you see and have a project you need coded,
-                     don’t hesitate to contact me.</p>
-        
-            </div>
-            </div>
-
-           
-           <div class="start-container"><button class="Start-collaboration"><p>Start collaboration</p></button></div> 
-            
-            
-          <div class="illustration-header" aria-hidden="true">
-            <img src="assets/" alt="">
+      <section class="home-container-desktop">
+        <div class="home_page home_page-desktop">
+          <div class="dot-container">
+            <h1>
+              THIERRY <br />
+              NOUMBUH
+            </h1>
+            <p class="intro_para">
+              Hello! I am a software developer! I can help you build a product,
+              feature or website. Take a look at my work.
+            </p>
+            <br />
+            <p class="intro_para">
+              If you like what you see and have a project you need coded, don’t
+              hesitate to contact me.
+            </p>
+            <button type="button" class="Start-collaboration">
+              <p>Start collaboration</p>
+            </button>
+            <div class="illustration-header" aria-hidden="true"></div>
           </div>
-        </section>
+        </div>
+        <img
+          src="assets/desktop-img/first-backgroung.svg"
+          class="illustration-header-desktop"
+          alt=""
+        />
+      </section>
 
-        <section class="project-section-container">
-            <div class="project-section">
-                <h2>projects</h2>
-            </div>
-        </section>
+      <section class="project-section-container">
+        <div class="project-section">
+          <h2>projects</h2>
+        </div>
 
-        <section class="containing-6">
-            <div class="rectangle_21">
-                <img src="assets/images/rectangle_21.svg" alt="rectangle_21" />
-            </div>
+        <div class="containing-6">
+          <div class="rectangle_21 item-1">
+            <img src="assets/images/rectangle_21.svg" alt="rectangle_21" />
             <div class="rectangle_22">
+              <div class="contianer-list">
                 <span class="project-name">project name goes here</span>
-                <div>
+                <ul class="project-languages">
+                  <li class="language-css">HTML/CSS</li>
+                  <li class="language-css">Ruby on Rails</li>
+                  <li class="language-css">JavaScript</li>
+                </ul>
+
+                <button type="button" class="project-button">
+                  <span>See this project</span>
+                  <img
+                    src="assets/images/arrow.svg"
+                    alt="arrow"
+                    class="arrow"
+                  />
+                </button>
```

Comment:
- [ ] Great job, @axcell-0!👍 I recommend maintaining consistency with the font size and weight to align with the Figma design. This will enhance the user experience and create a pleasant feel.😊 Additionally, please make sure the list of project languages is correctly aligned, and there should be a gap between the "See This Project" button.👍

Figma design:

![3-1](https://github.com/axcell-0/Thierrys-portfolio/assets/40009354/cc796c0e-3fc2-4920-951a-100c80f12395)

Your design:

![3](https://github.com/axcell-0/Thierrys-portfolio/assets/40009354/c6cae6c0-4192-45c9-bca2-853843c93051)